tests/rt.sh

```diff
@@ -207,9 +207,9 @@ elif [[ $MACHINE_ID = hera.* ]]; then
   ROCOTOCOMPLETE=$(which rocotocomplete)
   ROCOTO_SCHEDULER=slurm
 
-  PYTHONHOME=/scratch1/NCEPDEV/nems/emc.nemspara/soft/miniconda3_new_20210629
+  PYTHONHOME=/scratch1/NCEPDEV/nems/role.epic/miniconda3/4.12.0
```

Comment:
Revert this change to reproduce baseline

Reply:
done